### ISSUE TITLE
Converting Keras model to Caffe

### ISSUE THREAD

**Issue by SarthakYadav** [fchollet/keras#5232]
Hi Everyone.

I wanted to ask if there exists a way to convert existing Keras models to their Caffe counterpart.
The thing is, that the model we have developed is proprietary, and frankly, quite long and arduous to implement (in Keras itself to start with!). There are no custom loss functions or custom layers though and the team is wondering if writing the corresponding Caffe model is worth the time required (and the amount of protobuf to write!)
So if you could please point me towards a library/tool that can automate the conversion task, that would be great!
Thanks!!!

**Comment by rcox771** [comment]
Did you ever find an answer to your question? I'm trying to do the same thing.

**Comment by SarthakYadav** [comment]
Nope. I didn't find an answer. I simply learned Caffe as well ;)
